test_cases/iteration/iterate_urange.py

```diff
@@ -13,6 +13,7 @@ def test_forwards(self) -> None:
         for i in urange(1, 7, 2):
             log(values[i])
             i += 1
+        assert i == UInt64(6)
```

Comment:
Whenever there's a binary operation, as long as one operand type has an algopy type, the other operand can be the equivalent literal type. So this can just be `assert i == 6`.